service/bin/d-installer-users

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+#
+# Copyright (c) [2022] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+# TEMPORARY overwrite of Y2DIR to use DBus for communication with dependent yast modules
+ENV["Y2DIR"] = File.expand_path("../lib/dinstaller/dbus/y2dir", __dir__)
+
+$LOAD_PATH.unshift File.expand_path("../lib", __dir__)
+
+require "rubygems"
+require "bundler/setup"
+
+require "eventmachine"
+require "dinstaller/dbus/users_service"
+
+logger = Logger.new($stdout)
+
+EM.run do
```

Comment:
NP: We should use `EventMachine` everywhere for consistency

Reply:
well, it is just copied code from `bin/d-installer` so it is consistent with other service